NDTensors/src/empty/empty.jl

```diff
@@ -5,20 +5,20 @@
 
 struct EmptyOrder end
 
-function similartype(StoreT::Type{<:TensorStorage{EmptyNumber}}, ElT::Type)
+function similartype(StoreT::Type{<:TensorStorage{UnspecifiedZero}}, ElT::Type)
```

Comment:
I wonder why these were needed in the first place, seems like they should be covered by some generic `similartype` function and there's nothing special about `UnspecifiedZero` here.

Reply:
Though I guess these files aren't used any more and will be deleted, so it shouldn't matter.

Reply:
Yeah I am not sure, I did delete these files altogether and am working towards having all unit tests pass without them.